source/website/src/views/Step3.vue

```diff
@@ -265,7 +265,7 @@ SPDX-License-Identifier: Apache-2.0
         this.showMissingDataAlert = true
       }
       else if (!this.step3_form_input.length) {
-        this.send_request('POST', 'get_data_columns', {'s3bucket': this.DATA_BUCKET_NAME, 's3key':this.s3key, 'file_format':this.dataset_definition.fileFormat})
+        this.send_request('POST', 'get_data_columns', {'s3bucket': this.DATA_BUCKET_NAME, 's3key':this.s3key})
```

Comment:
I think we really need to surface errors from this request because I suspect people will often try to upload files with different schemas, and it will be difficult for them to know that the schemas are mismatched if we don't surface that error here.